tactical-microgrid-standard/common/Controller.cpp

```diff
@@ -24,14 +19,6 @@ DDS::ReturnCode_t Controller::init(DDS::DomainId_t domain, int argc, char* argv[
     return rc;
   }
 
-  tms::DeviceInfo device_info;
-  device_info.deviceId(device_id_);
-  device_info.role(tms::DeviceRole::ROLE_MICROGRID_CONTROLLER);
-  rc = send_device_info(device_info);
-  if (rc != DDS::RETCODE_OK) {
-    return rc;
-  }
-
   rc = start_heartbeats();
   if (rc != DDS::RETCODE_OK) {
     return rc;
```

Comment:
We should probably send device info before heart beats because the device announcement data flow is expected to be the first.

Reply:
Sure, but I'm not sure it's possible for it to be guaranteed to be received in that order though.